README.md

```diff
@@ -1,8 +1,6 @@
 # Kubot
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/kubot`. To experiment with that code, run `bin/console` for an interactive prompt.
-
-TODO: Delete this and the text above, and describe your gem
+Kubot is a Slack support bot that uses Slack API to foward messages from workspaces that are given support to workspace that provides it. It creates a database where all the workspaces registered are stored. And once Kubot is asked a question on channel in a supported workspace a corresponding channel will be created in support workspace and all the messages will be forwarded between this two channels. It's a helpful tool to manage your support workflow.
```

Comment:
foward -> forward.

Reply:
Bad word order: "all the workspace registered are stored" -> "all registered workspaces are stored"

Reply:
between this[..]channels, this -> these.

Reply:
> foward -> forward.

Don't quite understand what you mean by this